cmd/crc/cmd/root.go

```diff
@@ -96,15 +98,26 @@ func setProxyDefaults() {
 	httpProxy := config.GetString(cmdConfig.HTTPProxy.Name)
 	httpsProxy := config.GetString(cmdConfig.HTTPSProxy.Name)
 	noProxy := config.GetString(cmdConfig.NoProxy.Name)
+	proxyCAFile := config.GetString(cmdConfig.ProxyCAFile.Name)
+
+	// If proxyCAFile is not empty string then read the content and pass it to NewProxyDefaults
+	var proxyCACert []byte
+	if proxyCAFile != "" {
+		var err error
+		proxyCACert, err = ioutil.ReadFile(proxyCAFile)
+		if err != nil {
+			exit.WithMessage(1, fmt.Sprintf("not able to read proxyCAFile %s: %v", proxyCAFile, err.Error()))
+		}
+	}
 
-	proxyConfig, err := network.NewProxyDefaults(httpProxy, httpsProxy, noProxy)
+	proxyConfig, err := network.NewProxyDefaults(httpProxy, httpsProxy, noProxy, strings.TrimRight(string(proxyCACert), "\r\n"))
 	if err != nil {
 		exit.WithMessage(1, err.Error())
 	}
 
 	if proxyConfig.IsEnabled() {
-		logging.Debugf("HTTP-PROXY: %s, HTTPS-PROXY: %s, NO-PROXY: %s", proxyConfig.HTTPProxyForDisplay(),
-			proxyConfig.HTTPSProxyForDisplay(), proxyConfig.GetNoProxyString())
+		logging.Debugf("HTTP-PROXY: %s, HTTPS-PROXY: %s, NO-PROXY: %s, proxyCACert: %s", proxyConfig.HTTPProxyForDisplay(),
+			proxyConfig.HTTPSProxyForDisplay(), proxyConfig.GetNoProxyString(), proxyConfig.ProxyCACert)
```

Comment:
Isn't this the ProxyCACert too verbose for debug output? Maybe it's good enough to print only the filename *if we managed to read it successfully*?

Reply:
@cfergeau yes, it was my own debug purpose, have to add the filename only.